tests/data_validation/data_validation_list_component.test.ts

```diff
@@ -78,33 +79,13 @@ describe("Edit criterion in side panel", () => {
       ]);
     });
 
-    test("Can navigate values with keyboard", async () => {
```

Comment:
This was a really nice feature, I don't thinnk we want to remove it. And why would we repalce the input by a composer if the criterion only allow litteral values and no formulas ?